### DISCUSSION TITLE
suspicious site github.shali-sdc4.acirock.com

### DISCUSSION THREAD

**Discussion by vitusb** [community/community#79080] [Other Feature Feedback, Questions, & Ideas]
Hello,
i noticed a complete imitation of github.com, when looking for repo-names in Google:

> https://github.shali-sdc4.acirock.com

This site covers not the same ip address-space like "github.com". The most site-advisors are not listing this site as a phishing-site and allmost all black-lists don't have this site in their listings ... also Google does not list much infos about the domain ".acirock.com"

Does anybody has some infos about that site ?

**Comment by DnZmfr** [comment]
Indeed, seems to be a clone of https://github.com/finestructure
Google says the URL was first indexed on November 2023 (one month ago) and the acirock domain was registered on July 2020  🤔
<img width="1188" alt="image" src="https://github.com/community/community/assets/7255778/155eb0d0-f410-4465-ab95-3a127f381a07">

**Comment by astares** [comment]
Yes - google seems to suggest it as well already. Seems to be hosted on F5 Distributed Cloud Services.

They try to connect from Shreveport, LA, USA - still investigating

![image](https://github.com/community/community/assets/5980033/954a7f0a-c700-4b49-b901-9d281f34998d)

<img width="596" alt="image" src="https://github.com/community/community/assets/5980033/7f5e82b3-c0ff-4d1e-9ab6-c3634a024702">

**Comment by samus-aran** [comment]
Hi @vitusb , 👋🏻 We really appreciate you flagging this. The best route to get this to the proper GitHub team is to use our [abuse reporting tools](https://docs.github.com/en/communities/maintaining-your-safety-on-github/reporting-abuse-or-spam). Here's all the info:

- Reporting abuse or spam
- Reporting a user
- Reporting an issue or pull request
- Reporting a comment

You can report behavior and content that violates community guidelines and terms. We are going to close this post, but for this and any future incidents, please refer to the links above.

Thank you!
